spec/models/card_spec.rb

```diff
@@ -4,25 +4,55 @@
 DatabaseCleaner.strategy = :truncation
 
 describe Card do
-  before do
+  before(:all) do
     DatabaseCleaner.clean
     user = create(:user, email: "email@email.com", password: "****",
                          password_confirmation: "****")
     deck = user.decks.create(name: "Cat")
-    @card = deck.cards.create(translated_text: "text", original_text: "текст")
+    @card = deck.cards.create(translated_text: "text",
+                              original_text: "текст",
+                              user_id: user.id)
   end
 
   subject { @card }
 
   describe "has true data in model" do
-    it { expect(@card.review_date).to eq(Date.today + 3.days) }
+    it { expect(@card.review_date.to_i).to eq(DateTime.now.to_i) }
     it { expect(@card.translated_text).to eq("text") }
     it { expect(@card.original_text).to eq("текст") }
   end
 
-  describe "is valid check translation" do
+  describe "is valid first check translation" do
     it { expect(@card.check_translation("текст")).to be true }
-    it { expect(@card.review_date).to eq(Date.today + 3.days) }
+    it do
+      expect(@card.review_date.to_i)
+      .to eq(DateTime.current.to_i + 12.hours.to_i)
+    end
+  end
+
+  describe "is valid second check translation" do
+    it { expect(@card.check_translation("текст")).to be true }
+    it { expect(@card.review_date.to_i)
+         .to eq(DateTime.current.to_i + 3.day.to_i) }
+  end
+
+  describe "is valid third check translation" do
+    it { expect(@card.check_translation("текст")).to be true }
+    it { expect(@card.review_date.to_i)
+         .to eq(DateTime.current.to_i + 1.week.to_i) }
```

Comment:
Expression at 42, 54 should be on its own line.<br>Place the . on the previous line, together with the method call receiver.<br>Use 2 (not 5) spaces for indenting an expression spanning multiple lines.